Solutions/ApacheHTTPServer/Analytic Rules/ApacheCVE-2021-41773.yaml

```diff
@@ -0,0 +1,27 @@
+id: 767f9dc4-3b01-11ec-8d3d-0242ac130003
+name: Apache - Apache 2.4.49 flaw CVE-2021-41773 
+description: |
+  'Detects using Apache 2.4.49 flaw CVE-2021-41773 '
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ApacheHTTPServer
+    dataTypes:
+      - ApacheHTTPServer
+tactics:
+  - InitialAccess
+  - Lateral Movement

```

Comment:
LateralMovement should be one word without space.

Reply:
updated